keras/api/_tf_keras/keras/__init__.py

```diff
@@ -52,3 +47,16 @@
 from keras.src.regularizers.regularizers import Regularizer
 from keras.src.version import __version__
 from keras.src.version import version
+
+"""DO NOT EDIT.
```

Comment:
Nit: we should strip this block of comments (it appears 2x). Not important though.

Reply:
Done. We no longer have two comments section in the generated files of `_tf_keras`. This is also necessary to pass the formatter check.